packages/mobile/src/escrow/saga.ts

```diff
@@ -268,18 +267,19 @@ function* doFetchSentPayments() {
     const sentPaymentsRaw = yield all(
       sentPaymentIDs.map((paymentID) => call(getEscrowedPayment, escrow, paymentID))
     )
-    const tempAddresstoRecipientPhoneNumber: Invitees = yield select(inviteesSelector)
+
+    const addressToE164Number = yield select(addressToE164NumberSelector)
     const sentPayments: EscrowedPayment[] = []
     for (let i = 0; i < sentPaymentsRaw.length; i++) {
-      const id = sentPaymentIDs[i].toLowerCase()
-      const recipientPhoneNumber = tempAddresstoRecipientPhoneNumber[id]
+      const address = sentPaymentIDs[i]
```

Comment:
Should you be `toLowerCase`ing here?

Reply:
This was an open question for me. Should I be making `address` lowercase before attempting to look it up in `addressToE164Number`? And generally speaking, is the move to always convert to lowercase when dealing with addresses?

Reply:
Yeah that's recommended.

Reply:
Ok makes sense. In that case, I also needed to do it here: https://github.com/celo-org/celo-monorepo/pull/3612/files?file-filters%5B%5D=.json&file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx#diff-a4552fb871577da04d8ebdfed6dd9670R213